ui/src/groups/Groups.tsx

```diff
@@ -95,10 +94,18 @@ function Groups() {
   if (!connection && !group) {
     return (
       <div className="flex min-w-0 grow items-center justify-center bg-gray-50">
-        <div className="flex items-center justify-center">
+        <div className="flex flex-col items-center justify-center space-y-4">
           <span className="ml-2 text-gray-600">
             Group host ({ship}) is offline.
           </span>
+          <button
+            className="small-button"
+            onClick={() =>
+              useGroupConnectionState.getState().setGroupConnected(flag, true)
```

Comment:
what will this do exactly? so like we're in an "offline" state so we see that messaging, but if the group join goes through behind the scenes we'll have to hit this to see that?

Reply:
If we're in the "group is down" cached state, yeah.

Reply:
Yeah, good idea.